### ISSUE TITLE
[a11y]  - correct pronunciation of the scale value in the screen reader

### ISSUE THREAD

**Issue by transkamp** [open-pioneer/trails-openlayers-base-packages#432]
### Issue description

open https://open-pioneer.github.io/trails-demo/openlayers-base-packages/dev/samples/showcase/?demo=scaleSetter

when selecting the scale, the screen reader does not read it out correctly. the message should read 1:50,000 ==> scale 1 to 50000

In order for this to be implemented correctly, the speech output must be optimized with the aria-label attribute. aria-label=“Maßstab 1 to 50000”. It is important that the separators are not included in the aria-label. 

Otherwise the number will be read out incorrectly. The same must also be done in the scale selection list.

![Image](https://github.com/user-attachments/assets/36c0a834-629a-4ab0-bdad-771af92e5e3c)

`*aria-label="Aktueller Maßstab: 1 zu 272989"`

##############

the arrow must be turned 180 degrees. other dropdown elements have the arrow the other way around
![Image](https://github.com/user-attachments/assets/f9a46ca5-52a7-4302-a315-ee258d8319d7)

### Validations

- [x] Read and follow our [code of conduct](https://github.com/open-pioneer/.github/blob/main/CODE_OF_CONDUCT.md)
- [x] Read the [contributing guidelines](https://github.com/open-pioneer/trails-starter/blob/main/docs/Contributing.md)
- [x] Check that there isn't already a similar issue
